src/naming-strategy/DefaultNamingStrategy.ts

```diff
@@ -31,7 +31,7 @@ export class DefaultNamingStrategy implements NamingStrategyInterface {
             return customName;
 
         const key = "ind_" + tableName + "_" + columns.join("_");
-        return "ind_" + RandomGenerator.sha1(key).substr(0, 27);
+        return "in_" + RandomGenerator.sha1(key).substr(0, 27);
```

Comment:
why "in_"? If its about length then batter to change 27 to 26, because I would prefer "ind_" to stay same in all databases